srml/support/procedural/src/lib.rs

```diff
@@ -215,3 +216,63 @@ use proc_macro::TokenStream;
 pub fn decl_storage(input: TokenStream) -> TokenStream {
 	storage::transformation::decl_storage_impl(input)
 }
+
+/// Construct a runtime, with the given name and the given modules.
+///
+/// The parameters here are specific types for `Block`, `NodeBlock`, and `InherentData`
+/// and the modules that are used by the runtime.
+/// `Block` is the block type that is used in the runtime and `NodeBlock` is the block type
+/// that is used in the node. For instance they can differ in the extrinsics type.
+///
+/// # Example:
+///
+/// ```nocompile
```

Comment:
Please make this compile.

Reply:
@bkchr I tried to make it compile, but it has some obstacles:
Since the macro expansion relies on having `srml-support` crate in dependencies, it looks like it's impossible to have this doc test in `srml-support-procedural` crate (adding `srml-support` to `srml-support-procedural` will have a circular dependency). We can move the docs to `srml-support` crate but then `srml-support-procedural::construct_runtime` will be undocumented or having a duplicate docs which is not very nice also. To me it looks like it's not worth of having the doc test, but rather leave the nocompile option as it is.